packages/theme-check-common/src/checks/liquid-html-syntax-error/index.ts

```diff
@@ -96,7 +97,10 @@ export const LiquidHTMLSyntaxError: LiquidCheckDefinition = {
 
           context.report(problem);
         },
-        async LiquidVariableOutput(node) {
+        async LiquidVariableOutput(node, ancestors) {
+          // Do not report {{ foo == bar }} errors inside {% schema %} where they are supported
+          if (isWithinRawTagThatDoesNotParseItsContents(ancestors)) return;
```

Comment:
<img width="894" height="254" alt="Image" src="https://github.com/user-attachments/assets/a358e516-6d8f-43f3-a8a5-a083d61ee2ca" />

That's a false positive